vspace/src/main/java/edu/asu/diging/vspace/core/model/ISlide.java

```diff
@@ -0,0 +1,7 @@
+package edu.asu.diging.vspace.core.model;
+
+
+public interface ISlide {
+	
+	
+}
```

Comment:
doesn't seem right that this is empty

Reply:
all the properties of Slide - title, description, and image have been declared in other interfaces. Do you want me to remove this interface in that case?

Reply:
your implementaiton of ISlide has getters and setters for an image though, doesn't it?

Reply:
also the interface doesn't extends IVSpaceElement